docs/pages/desktop-access/rbac.mdx

```diff
@@ -123,6 +129,79 @@ allow:
   windows_desktop_logins: ["{{internal.windows_logins}}"]
 ```
 
+### Automatic User Creation
+
+<Details title="Version Warning: Before 12.3" opened={false}>
+Teleport versions prior to 12.3 don't support the options shown below.
+</Details>
+
+Teleport's Desktop Service can be configured to automatically create local
+Windows users upon login.
+
+#### Configuration
+
+<Admonition type="warning" title="Local users only">
+Automatic user provisioning is only supported for local users, and does
+not take effect in Active Directory environments.
+</Admonition>
+
+This feature is disabled by default, and can be enabled by setting the
+`create_desktop_user` role option on one of the user's roles.
+
+```yaml
+kind: role
+version: v6
+metadata:
+  name: allow-user-provisioning
+spec:
+  options:
+    create_desktop_user: true
+  allow:
+    windows_desktop_labels: { '*': '*' }
+    windows_desktop_logins: jane
+```
+
+When a Teleport user connects to a desktop, Teleport checks each of the user's
+roles that match the desktop. If at least one role matches the desktop but does
+not include `create_desktop_user: true`, automatic user creation will be
+disabled. Roles that do not match the Node will not be checked.
+
+In order to create the user, the requested username must be present in one of the
+role's `windows_desktop_logins`.
+
+#### User management
+
+By default, the newly-created user will be placed in two Windows groups:
+
+- `Remote Desktop Users`: a built-in Windows group allowing remote desktop access
+- `Teleport Users`: a custom group that Teleport places all
+  automatically-created users in
+
+To add the user to additional groups, specify the `desktop_groups` role option:
+
+```yaml
+kind: role
+version: v6
+metadata:
+  name: allow-user-provisioning
+spec:
+  options:
+    create_desktop_user: true
+  allow:
+    windows_desktop_labels: { '*': '*' }
+    windows_desktop_logins: jane
+    desktop_groups:
+    - developers
```

Comment:
Do we want to mention explicitly that if you want user to be an administrator you should add it to `Administrators` group?

Reply:
Sure, will do